File: 9.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
  if (x < 0) {
    return false
  }
  if (x < 10) {
    return true
  }
  const s = String(x)
  for (let i = 0; i < s.length / 2; i++) {
    if (s[i] !== s[s.length - i - 1]) {
      return false
    }
  }
  return true
}

File: 9.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0:
            return False
        if x < 10:
            return True
        s = str(x)
        for i in range(len(s) // 2):
            if s[i] != s[-i-1]:
                return False
        return True
